docs/product/components/usercard.html

```diff
@@ -0,0 +1,189 @@
+---
+layout: page
+title: User Cards
+description: User cards are a combination of a user and metadata about the user or post
+---
+
+<section class="stacks-section">
+    {% header "h2", "Regular" %}
+    <div class="stacks-preview">
+{% highlight html %}
+
+    <div class="s-usercard">
+        <time class="s-usercard--time">3 mins ago</time>
+    
+        <a href="#" class="s-usercard--avatar s-avatar s-avatar__md">
+            <img src="https://placekitten.com/32/32" />
+        </a>
+    
+        <a href="#">
+            <h4 class="s-usercard--name">Paul Wright</h4>
+        </a>
+    
+        <ul class="list-reset s-usercard--awards">
+            <li class="s-usercard--rep">
+                2,500k
+            </li>
+    
+            <li class="s-award-bling s-award-bling__gold">
+                5
+            </li>
+    
+            <li class="s-award-bling s-award-bling__silver">
+                9
+            </li>
+    
+            <li class="s-award-bling s-award-bling__bronze">
+                1
+            </li>
+        </ul>
+    </div>
+
```

Comment:
```suggestion
```

Reply:
Same as previous - I can't see anything except "-" in this suggested change. Going to assume it's whitespace which I fixed